bin/middlewares/unsupported_browser.js

```diff
@@ -0,0 +1,17 @@
+/* globals module */
+
+function isUnsupported(userAgent = '') {
+  return userAgent.match('Trident'); // old IE detection
+}
+
+module.exports = function(config) {
+  const redirect = config.server.unsupportedBrowsers.redirect;
+
+  return function(req, res, next) {
+    if (redirect && isUnsupported(req.headers['user-agent'])) {
+      res.redirect(302, req.baseUrl + '/unsupported');
```

Comment:
`req.baseUrl` is the base URL used by the parent router. In practice on deployed environments, Erdapfel is behind a proxy that is responsible for the routing of URLs among all maps services. So `req.baseUrl` would be empty in this case, even if the user needs to be redirected to `/maps/unsupported`.

`config.system.baseUrl` should probably be used instead of `req.baseUrl`:
```suggestion
      res.redirect(302, config.system.baseUrl + 'unsupported');
```
```